Documentation/reference/os/iob.rst

```diff
@@ -164,6 +164,7 @@ Public Function Prototypes
   - :c:func:`iob_remove_queue()`
   - :c:func:`iob_peek_queue()`
   - :c:func:`iob_free_queue()`
+  - :c:func:`iob_free_qentry_from_queue()`
```

Comment:
@antmerlino and @anchao what about we name the new API iob_free_qentry directly without _from_queue suffix?